geth-dev-node/start.sh

```diff
@@ -1,3 +1,2 @@
-geth --password /app/password.txt account new
-geth init /app/genesis.json
-geth --password /app/password.txt --unlock 0 --mine --networkid=10000 --ws --wsapi eth,net,web3,personal --wsport 8546 --rpc --rpcapi eth,net,web3,personal,miner --rpcaddr 0.0.0.0 --targetgaslimit 6700000 --ethash.dagdir "/app/.ethash"
+geth --datadir /home/\.ethereum init /home/genesis.json
+geth --datadir /home/\.ethereum --keystore /home/\.ethereum/keystore --password /home/password.txt --unlock "0xc5ed899b0878656feb06467e2e9ede3ae73cbcb7" --mine --ws --wsapi eth,net,web3,personal --wsport 8546 --rpc --rpcapi eth,net,web3,personal,miner --rpcaddr 0.0.0.0 --targetgaslimit 6500000
```

Comment:
Parity lets you have a config file rather than configuring Parity exclusively via CLI arguments.  Does Geth have something similar or is CLI arguments the only way to configure it?

Reply:
I'm not sure -- I'll check on that.  I only did it this was because this was the only way I'd seen it done elsewhere.